data_validation/cli_tools.py

```diff
@@ -988,9 +988,9 @@ def list_connections():
     """List all saved connections."""
     mgr = state_manager.StateManager()
     connections = mgr.list_connections()
-
     for conn_name in connections:
-        logging.info(f"Connection Name: {conn_name}")
+        source_type = mgr.get_connection_config(conn_name).get("source_type")
```

Comment:
Should we also add target_type along with source? @nehanene15 any thoughts?

Reply:
@Raniksingh 
AFAIU, connection listing implies databases whose connection details is added,  we can't confirm whether it is used as target when using DVT. source_type itself should suffice, it says to which source\database connection is pointing to.